test/test.py

```diff
@@ -96,7 +90,8 @@ def _test_integration(descr, args, ref):
     _test_integration('Project config not found',
                       ['--config', 'notfound.json', 'update'],
                       'Project config file \'notfound.json\' not found. Exiting.\n')
-    _test_integration('Create command', ['--auth', 'auth.json', 'create'], 'log.csv is created\n')
```

Comment:
Is it intentional `--auth` => `--config`?

Reply:
Yes. Config file is 'config.json' from the current folder by default. After this restructure we don't have it any longer, but have 'examples/config_personal.json' instead. So, have to provide '--config' option to the integration test, pointing there.
Having or not '--auth' piece doesn't really mean anything yet. I had it before to hit code coverage target :) I'll add Search Engine Optimization tracking metric soon, this is where '--auth' option will really kick in as users will need to provide their Google keys with it.